software/huly.yml

```
@@ -0,0 +1,27 @@
+name: "Huly"
+website_url: "https://huly.io"
+source_code_url: "https://github.com/hcengineering/platform"
+description: "All-in-One Project Management Platform (alternative to Linear, Jira, Slack, Notion, Motion)."
+licenses:
+  - EPL-2.0
+platforms:
+  - Docker
+  - K8S
+  - Nodejs
+  - Javascript
+tags:
+  - Software Development - Project Management
+  - Automation
+  - Calendar & Contacts
+  - Communication - Custom Communication Systems
+  - Document Management
+  - Groupware
+  - Human Resources Management (HRM)
+  - Knowledge Management Tools
+  - Note-taking & Editors
+  - Task Management & To-do Lists
+  - Ticketing
+  - Time Tracking
+  - Wikis
```

Please don't try to abuse the tagging system...

```suggestion
tags:
  - Software Development - Project Management
```